### PR TITLE
Fix few problems

### PR DESCRIPTION
**Pull request by szepeviktor**
I used a unnamed tool.

### CODE REVIEW

File: src/commands/outdated/OutdatedCommand.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ private function renderCliOutput(array $outdated): string {
 
         return \sprintf(
             "Found %d outdated PHARs in phive.xml:\n\n%s",
-            $outdated,
+            count($outdated),
```
Comment (szepeviktor) [inline src/commands/outdated/OutdatedCommand.php:76]: @theseer This is critical.

File: src/autoload.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // @codeCoverageIgnoreStart
 // this is an autogenerated file - do not edit
 spl_autoload_register(
-    function($class) {
+    function(string $class): void {
```
Comment (theseer) [inline src/autoload.php:6]: What does "This is an autogenerated file - do not edit" tell you? ;)

File: src/commands/outdated/OutdatedCommand.php (hunk 1 above)
Comment (theseer) [reply src/commands/outdated/OutdatedCommand.php:76]: Already fixed locally as psalm complained as well ;)

File: src/services/key/gpg/GnupgKeyImporter.php [hunk 3]
```diff
@@ -2,10 +2,10 @@
 namespace PharIo\Phive;
```
Comment (theseer) [inline src/services/key/gpg/GnupgKeyImporter.php:3]: Already fixed these locally. Thanks.

File: src/services/key/gpg/PublicKeyReader.php [hunk 4]
```diff
@@ -16,7 +16,7 @@ class PublicKeyReader {
     /**
      * PublicKeyReader constructor.
      */
-    public function __construct(Gnupg $gnupg, Directory $workDir) {
+    public function __construct(GnuPG $gnupg, Directory $workDir) {
```
Comment (theseer) [inline src/services/key/gpg/PublicKeyReader.php:19]: Already fixed this locally. Thanks.

File: src/services/signature/gpg/GnupgSignatureVerifier.php [hunk 5]
```diff
@@ -5,13 +5,13 @@
  * GPG Signature Verification using the GnuPG PECL Extension.
  */
 class GnupgSignatureVerifier implements SignatureVerifier {
-    /** @var Gnupg */
+    /** @var GnuPG */
     private $gpg;
```
Comment (theseer) [inline src/services/signature/gpg/GnupgSignatureVerifier.php:9]: Already fixed this locally. Thanks.

File: src/shared/GnuPG.php [hunk 6]
```diff
@@ -5,13 +5,13 @@
 
 class GnuPG {
 
-    /** @var \GnuPG */
+    /** @var \gnupg */
     private $gnupg;
```
Comment (theseer) [inline src/shared/GnuPG.php:9]: Already fixed these locally. Thanks.

File: src/shared/environment/UnixoidEnvironment.php [hunk 7]
```diff
@@ -25,7 +25,7 @@ public function hasHomeDirectory(): bool {
     }
 
     /**
-     * @throws DirectoryException
+     * @throws \PharIo\FileSystem\DirectoryException
      * @throws \BadMethodCallException
```
Comment (theseer) [inline src/shared/environment/UnixoidEnvironment.php:29]: Also have this locally.

**Comment by theseer** [comment]
You were to early with this it seems. I don't think there's anything left that I didn't have already ;)

**Pull request closed without merging**